litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx

```diff
@@ -60,7 +60,7 @@ const PanelContent: React.FC<GraphPanelProps> = ({
   const lineGraph: string[] = palette.graph.line;
   const areaGraph: string[] = palette.graph.area;
   const [popout, setPopout] = useState(false);
-  const [viewEventMetric, setViewEventMetric] = useState(false);
+  const [viewGraphMetric, setViewGraphMetric] = useState(false);
   const [prometheusQueryData, setPrometheusQueryData] =
```

Comment:
while replacing the `EventMetric` export with `GraphMetric` the `viewEventMetric` hook also got changed. May we please revert it to `veiwEventMetirc`

Reply:
Yes both were separate @S-ayanide

Reply:
Yes sure, reverting this change